packages/hardhat/deploy/00_deploy_your_contract.ts

```diff
@@ -21,22 +21,54 @@ const deployYourContract: DeployFunction = async function (hre: HardhatRuntimeEn
   const { deployer } = await hre.getNamedAccounts();
   const { deploy } = hre.deployments;
 
-  await deploy("YourContract", {
+  await deploy("Balloons", {
     from: deployer,
     // Contract constructor arguments
-    args: [deployer],
+    //args: [deployer],
     log: true,
     // autoMine: can be passed to the deploy function to make the deployment process faster on local networks by
     // automatically mining the contract deployment transaction. There is no effect on live networks.
     autoMine: true,
   });
-
-  // Get the deployed contract
+// Get the deployed contract
   // const yourContract = await hre.ethers.getContract("YourContract", deployer);
+  const balloons = await hre.ethers.getContract("Balloons", deployer);
+
+  await deploy("DEX", {
+    from: deployer,
+    // Contract constructor arguments
+    args: [balloons.address],
+    log: true,
+    // autoMine: can be passed to the deploy function to make the deployment process faster on local networks by
+    // automatically mining the contract deployment transaction. There is no effect on live networks.
+    autoMine: true,
+  });
+
+  const dex = await hre.ethers.getContract("DEX", deployer);
+
+    // paste in your front-end address here to get 10 balloons on deploy:
```

Comment:
this block should also be commented https://github.com/scaffold-eth/se-2-challenges/pull/15/files#r1269525687